microsoft-edge/webview2/roadmap.md

```diff
@@ -23,32 +23,16 @@ If you have concerns or questions about the Roadmap, provide your feedback in th
 
 The WebView2 team is planning the following major efforts for future updates.  
 
-:::row:::
-   :::column span="1":::
-      WebView2 Runtime Installer  
-   :::column-end:::
-   :::column span="2":::
-      *   Q4 2020
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      Fixed Version  
-   :::column-end:::
-   :::column span="2":::
-      *   Q4 2020  
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      General Availability  
-   :::column-end:::
-   :::column span="2":::
-      *   Win32 C/C++ \(Q4 2020\)  
-      *   .NET \(Q4 2020\)  
-      *   [WinUI 3.0][GithubMicrosoftUiXamlRoadmap]  
-   :::column-end:::
-:::row-end:::  
+H2 2021:
+* Desktop UWP Preview
+
+H1 2022:
+* Desktop UWP GA
+
+H2 2022 & Beyond:
+* MacOS Preview
+* Xbox Preview
+* HoloLens Preview
```

Comment:
@jasonstephen15  - re: MacOS, Xbox, and HoloLens - are these related to UWP releases?

Reply:
No, they are seperate releases we don't have too much more specifc info about